### ISSUE TITLE
Revision 231 doesn't do anything, 0.7 can't open stdin

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#146]
**Reported by dfnsonfsduifb on 20 Mar 2007 15:29 UTC**

I'm on an AMD64 system (x86_64) here and am trying to get aircrack-ng running. With 0.7 I ran into a problem with opening stdin:

```
# ./aircrack-ng -w - -e fhdsui ../out.pcap 
fopen(dictionary) failed: Bad file descriptor
```

It fails in line 2283 to open the file descriptor. I don't know why 0 isn't a valid description for stdin in this case. Anyways I don't see why the already open "stdin" variable (which is of type FILE*) isn't used but fdopen is used instead. However, simply replacing the whole thing by "opt.dict = stdin" didn't do the trick - probably the descriptor is closed somewhere earlier on, I didn't investigate further.

Then I tried the SVN version (rev 231) which seems completely broken - it just doesn't do anything:

```
# ./aircrack-ng -w - -e foobar ../out.pcap
# ./aircrack-ng -w dict ../out.pcap
```

However, when not specifying a dictionary:

```
# ./aircrack-ng ../out.pcap
Opening ../out.pcap
Read 1339199 packets.

   #  BSSID              ESSID                     Encryption

   1  00:xx:xx:xx:xx:xx  xxxxxxxx                  None (10.x.x.x)
   [...]
```

It then asks me for the index number and after entering that obviously aborts because I haven't specified any dictionary.

**Comment by aircrack-ng** [comment]
**Comment by hirte on 20 Mar 2007 15:29 UTC**

(In #232) Fixed stupid {} bug :/ (Closes #146).

**Comment by aircrack-ng** [comment]
**Modified by misterx on 20 Mar 2007 15:29 UTC**
